From 684cf85eeac78fd716f476a6b9678cc11336f8b6 Mon Sep 17 00:00:00 2001
From: "Brian C. Lane" <bcl@redhat.com>
Date: Wed, 5 Dec 2018 08:52:51 -0800
Subject: [PATCH] Rename package to gnupg1 (#1656282)

gnupg2 is going to start using /usr/bin/gpg so we need to rename the
gnupg v1.4.x version to avoid conflicts.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index e5bf1bc1a..085ebd33e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -42,7 +42,7 @@ m4_define([mym4_betastring],
 m4_define([mym4_isgit],m4_if(mym4_betastring,[],[no],[yes]))
 m4_define([mym4_full_version],[mym4_version[]mym4_betastring])
 
-AC_INIT([gnupg],[mym4_full_version], [https://bugs.gnupg.org])
+AC_INIT([gnupg1],[mym4_full_version], [https://bugs.gnupg.org])
 
 
 development_version=mym4_isgit
-- 
2.19.2

